src/lib/services-manager/widgets/start_stop_button.rb

```diff
@@ -47,6 +47,18 @@ def widget
         PushButton(id, label)
       end
 
+      # Help text
+      #
+      # @return [String]
+      def help
+        # TRANSLATORS: help text for the 'Start'/'Stop' button
+        _(
+          "<b>Start</b> sets the service to be started (only shown if the service is inactive)." \
```

Comment:
This text is unclear to me. Normally, a "start" button starts something immediately when you click on it. This help text, however, sounds like this only enables starting the service, so it will be started at some later point (when?).

Which one is it? When I ask myself that question, it is likely that a normal user will also be confused.

Reply:
Yes, in Services Manager nothing happens on the fly. To clarify it, a Note is added at the end (you commented about that Note just above).